src/command_helpers/asset_helper.rb

```diff
@@ -46,6 +46,13 @@ def raise_error_if_node_is_missing
           Can not find node: #{options.node}
         EOF
       end
+
+      def error_if_asset_file_doesnt_exist(asset_name, asset_path)
```

Comment:
Both the `asset_name` and `asset_path` are not require as inputs to this method. The `asset_path` can be found from the name alone.